rezerwacje-app/src/main/java/org/niewidoczniakademicy/rezerwacje/core/model/Room.java

```diff
@@ -0,0 +1,29 @@
+package org.niewidoczniakademicy.rezerwacje.core.model;
+
+import com.opencsv.bean.CsvBindByName;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.lang.NonNull;
+
+import javax.persistence.*;
+
+@Entity
+@Table(uniqueConstraints={
+        @UniqueConstraint(columnNames = {"building", "name"})
```

Comment:
Tu bym to chyba zastapil po prostu adnotacjami dotyczacymi constraintow nad odpowiednimi polami

Reply:
Tutaj się chyba nie da, chodziło mi o to żeby pary (budynek, sala) się nie powtarzały, ale sam budynek będzie się powtarzał, nazwa sali też może.